### PR TITLE
fix Issue 14881 - [REG] posix.mak omits package.d files when building…

### PR DESCRIPTION
**Pull request by WalterBright**
… zip file

https://issues.dlang.org/show_bug.cgi?id=14881

### CODE REVIEW

File: posix.mak [hunk 1]
```diff
@@ -374,7 +374,9 @@ clean :
 	rm -rf $(ROOT_OF_THEM_ALL) $(ZIPFILE) $(DOC_OUTPUT_DIR)
 
 zip :
-	zip $(ZIPFILE) $(MAKEFILE) $(ALL_D_FILES) $(ALL_C_FILES) index.d win32.mak win64.mak osmodel.mak
+	zip $(ZIPFILE) $(MAKEFILE) $(ALL_D_FILES) $(ALL_C_FILES) index.d win32.mak win64.mak osmodel.mak \
+	std/range/package.d std/algorithm/package.d std/container/package.d std/regex/package.d \
+	std/experimental/logger/package.d
```
Comment (MartinNowak) [inline posix.mak]: We already have a variable for all packages, please use `$(addsuffix /package.d,$(STD_PACKAGES))` instead.

**Comment by MartinNowak** [comment]
I'm clueless why you need a zip package, but in the long-term a `zip $(ZIPFILE) $$(git ls-files)` based rule will require led maintenance.

File: posix.mak (hunk 1 above)
Comment (WalterBright) [inline posix.mak]: done

**Comment by WalterBright** [comment]
> I'm clueless why you need a zip package,

To transfer to systems that don't have git installed on them.

**Comment by MartinNowak** [comment]
This should have gone into stable.

**Comment by MartinNowak** [comment]
a563512c890f0b64354ed12b856dcb490652c53f

**Pull request merged**